custom/icds_reports/ucr/data_sources/dashboard/commcare_user_cases.json

```diff
@@ -38,6 +38,19 @@
           "property_name": "owner_id"
         }
       },
+      {
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "expression": {
+          "location_id_expression": {
+            "type": "property_name",
+            "property_name": "owner_id"
+          },
+          "type": "location_parent_id"
```

Comment:
Using this, as there is one lookup, assuming the owner_id is AWC

Reply:
@emord surely this could be a user at any level of the hierarchy? In code it looks like we always assume it's an AWW but I don't see anywhere that would restrict user cases to only AWWs

Reply:
Yeah. I don't see anywhere where we limit what user shows up in this table.

However the only place this is currently used looks to be https://github.com/dimagi/commcare-hq/blob/je/test-aaa-aggregation/custom/icds_reports/utils/aggregation_helpers/awc_location.py#L84-L94 where we join on an awc_id. So its possible that this has never been a problem because of that. 

I don't see any issue with adding something to the `configured_filter` that limits it to AWW (though we may need to add support for other workers later)

Reply:
I couldn't find any potential filter looking at a [sample doc](https://www.icds-cas.gov.in/hq/admin/doc_in_es/?id=1c801b8660294f66a051f4ce8ab69d2f) to add to `configured_filter`.

Reply:
I think let's revert this one for now. The expression as you have it will work but it will be null for users who aren't AWWs which means we can't shard it.

Reply:
You mean, skip adding supervisor_id to this particular UCR?

Reply:
yes

Reply:
Updated

Reply:
@sravfeyn you can use `commcare_location_id` to get the location's id and get the type of location from a related doc lookup

Reply:
Ha, cool. I have removed this UCR altogether per Simon's comment above.